sdk/appconfiguration/Azure.Data.AppConfiguration/tests/ConfigurationLiveTests.cs

```diff
@@ -1692,6 +1698,7 @@ public async Task CanModifyTheFilterParameterValues()
         }
 
         [RecordedTest]
+        [ServiceVersion(Min = ConfigurationClientOptions.ServiceVersion.V2023_10_01)]
```

Comment:
You don't have to add these attributes here, we can use the default latest version.

Reply:
We need to have those now to ensure the test won't run in 1.0 (which would make it fail).

Reply:
Yes, I thought we would remove the last GA version.